src/core/device.c

```diff
@@ -176,6 +176,7 @@ static int device_coldplug(Unit *u) {
 }
 
 static void device_catchup(Unit *u) {
+        Manager *m = u->manager;
```

Comment:
hmm, can you elaborate why you need to know at all whether the udev db is trusted or not?

Reply:
It's supposed to be describe in the commit message. Which part of it should be improved ?